build/scripts/test-determinism.ps1

```diff
@@ -12,26 +13,25 @@ $ErrorActionPreference = "Stop"
 $script:skipList = @()
 
 # Location that deterministic error information should be written to. 
-[string]$script:errorDir = ""
-[string]$script:errorDirLeft = ""
-[string]$script:errorDirRight = ""
+[string]$errorDir = ""
+[string]$errorDirLeft = ""
+[string]$errorDirRight = ""
 
-function Run-Build([string]$rootDir, [switch]$restore = $false, [string]$logFile = $null) {
+function Run-Build([string]$rootDir, [string]$logFile = $null) {
     Push-Location $rootDir
     try {
 
         # Clean out the previous run
         Write-Host "Cleaning the Binaries"
-        Exec-Console $msbuild "/nologo /v:m /nodeReuse:false /t:clean Roslyn.sln" 
+        Remove-Item -Recurse (Get-ConfigDir $rootDir) 
+        Remove-Item -Recurse (Get-ObjDir $rootDir) 
```

Comment:
@sharwell this is the clean you're looking for. Before every build the entire Obj and bin directories are nuked. Previously I was using `/t:clean` and that's woefully insufficient. It barely deletes anything.